### DISCUSSION TITLE
My python repo is mislabel to html

### DISCUSSION THREAD

**Discussion by caphefalumi** [community/community#69285] [Repositories] [answered]
### Select Topic Area

Question

### Body

The problem at hand is that my repository, which is primarily Python-based, has been mislabeled as HTML. This looks kinda annoying, here is my repo link:https://github.com/Dangduytoan12l/WordToExcel, anyone know how to fix this?

**Comment by aelew** [comment]
GitHub labels repositories using the most commonly used language, and it seems like you have a large [xref-app.html](https://github.com/Dangduytoan12l/WordToExcel/blob/main/build/app/xref-app.html) file in your repository. Getting rid of this file will probably solve your issue.

**Comment by haitham-alabdullah** [comment, marked as answer]
Make .gitattributes file in the root if not exist,
add these two lines at the end.

*.* linguist-detectable=false
*.py linguist-detectable=true

**Reply by haitham-alabdullah** [reply]
I make a pull request to your repo to fix this issue, you can just merge it

**Reply by caphefalumi** [reply]
Thanks a lot
